examples/radiotoolbutton.py

```diff
@@ -0,0 +1,47 @@
+from gi.repository import Gtk
+
+from sugar3.graphics.radiotoolbutton import RadioToolButton
+from sugar3.graphics.radiopalette import RadioPalette, RadioMenuButton
+from sugar3.graphics.xocolor import XoColor
+from sugar3.graphics import style
+
+
+window = Gtk.Window()
+window.show()
+window.connect("destroy", Gtk.main_quit)
+
+box = Gtk.HBox()
+window.pack_start(box, True, True, 0)
+box.show()
+
+
+def echo(button, label):
+    if not button.props.active:
+        return
+    print label
+
+
+palette = RadioPalette()
+# Adding 3 RadioToolButtons to a palette
+
+button1 = RadioToolButton(icon_name='document-save', accelerator="<ctrl>S", /
+                          xo_color=XoColor("#FF0000,#00FF00"))
+button1.connect('clicked', lambda button: echo(button, 'document-save'))
```

Comment:
Isn't there a special signal that is used for the radiotoolbuttons that detects when they are toggled or sonething?

Have a look at https://github.com/samdroid-apps/sugar/blob/help-nav-buttons/src/jarabe/view/viewhelp.py#L307